File: api/v2010/trigger/create-default/create-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

trigger = @client.usage.triggers
                       .create(
                          callback_url: 'https://example.com',
                          trigger_value: 'trigger_value',
                          usage_category: 'answering-machine-detection'
                        )

puts trigger.sid

File: api/v2010/trigger/create-default/create-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Usage;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var trigger = TriggerResource.Create(
            callbackUrl: new Uri("https://example.com"),
            triggerValue: "TriggerValue",
            usageCategory: TriggerResource.UsageCategoryEnum.AnsweringMachineDetection
        );

        Console.WriteLine(trigger.Sid);
    }
}
